readme.md

```diff
@@ -120,6 +120,13 @@ Force enable/disable the spinner. If not specified, the spinner will be enabled
 
 Note that `{isEnabled: false}` doesn't mean it won't output anything. It just means it won't output the spinner, colors, and other ansi escape codes. It will still log text.
 
+##### consumeStdin
+
+Type: `boolean`<br>
+Default: `false`
+
+Consume stdin input(except Ctrl+C) while running if it is TTY. Setting it to `true` will prevent spinner from twitching on input, outputting broken lines on `enter` presses, and prevent buffering of input for following reads or commands run after current process exits while spinner is running.
```

Comment:
> and prevent buffering of input for following reads or commands run after current process exits while spinner is running.

I don't think this is useful. The point of this text is to show the benefits of this option.

Reply:
I'd say it is the most beneficial part actually. Displaying entered text just breaks the rendering, while buffering input may lead to much more severe problems. Try running a spinner from master and typing `sudo rm -rf /*[Enter]` while it's running. (Don't.)

Also, it is a hard to avoid side effect, so should be explained IMO.

Reply:
Then I think it at least could be written clearer.

> input for following reads

Not clear what this means.